fzf.el

```diff
@@ -44,7 +44,7 @@
   "Configuration options for fzf.el"
   :group 'convenience)
 
-(defcustom fzf/window-height 15
+(defcustom fzf/window-height 10
```

Comment:
can you keep this at 15?  also, if you can squash everything to a single commit that'd be great.  thanks.